### PR TITLE
Move changelog for failed backport 1836

### PR DESCRIPTION
**Pull request by duytnguyendtn**
### Description

Switched the changelog entry to 3.2 for #1836 after a failed backport: https://github.com/spacetelescope/jdaviz/pull/1836#issuecomment-1313975699

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Review by kecnry — APPROVED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -26,6 +26,8 @@ Mosviz
 
 - Added coordinates display panels for Mosviz viewers. [#1795]
 
+- ``mosviz_row`` metadata now included in NIRISS-parsed 1D spectra. [#1836]
```
Comment (pllim) [inline CHANGES.rst:29]: Shouldn't this be in the bugfix subsection of 3.2?

Comment (kecnry) [reply CHANGES.rst:29]: woops, good catch!  Or maybe it could be merged with the changelog entry of #1790 if the bug wasn't exposed until then?

Comment (pllim) [reply CHANGES.rst:29]: Re: merging -- I'll leave that to your good judgement.

Comment (duytnguyendtn) [reply CHANGES.rst:29]: Hmm... Good points! Let me decide where to move it and I'll push again!

**Review by pllim — APPROVED** [review]
Fine by me if @kecnry ok with not merging it.

**Comment by duytnguyendtn** [comment]
Thanks everyone!

**Pull request merged**